test/lib/pgBackRestTest/Module/Mock/MockArchiveStopTest.pm

```diff
@@ -44,14 +44,14 @@ sub run
 
     foreach my $rhRun
     (
-        {vm => VM1, remote => false, s3 => false, encrypt => false, compress =>  true, error => 0},
-        {vm => VM1, remote =>  true, s3 =>  true, encrypt =>  true, compress => false, error => 1},
-        {vm => VM2, remote => false, s3 =>  true, encrypt => false, compress => false, error => 0},
-        {vm => VM2, remote =>  true, s3 => false, encrypt =>  true, compress =>  true, error => 0},
-        {vm => VM3, remote => false, s3 => false, encrypt =>  true, compress => false, error => 0},
-        {vm => VM3, remote =>  true, s3 =>  true, encrypt => false, compress =>  true, error => 1},
-        {vm => VM4, remote => false, s3 =>  true, encrypt =>  true, compress =>  true, error => 0},
-        {vm => VM4, remote =>  true, s3 => false, encrypt => false, compress => false, error => 0},
+        {vm => VM1, remote => false, s3 => false, encrypt => false, compress =>    GZ, error => 0},
+        {vm => VM1, remote =>  true, s3 =>  true, encrypt =>  true, compress =>    GZ, error => 1},
+        {vm => VM2, remote => false, s3 =>  true, encrypt => false, compress => undef, error => 0},
+        {vm => VM2, remote =>  true, s3 => false, encrypt =>  true, compress =>    GZ, error => 0},
+        {vm => VM3, remote => false, s3 => false, encrypt =>  true, compress => undef, error => 0},
+        {vm => VM3, remote =>  true, s3 =>  true, encrypt => false, compress =>    GZ, error => 1},
+        {vm => VM4, remote => false, s3 =>  true, encrypt =>  true, compress =>    GZ, error => 0},
+        {vm => VM4, remote =>  true, s3 => false, encrypt => false, compress => undef, error => 0},
```

Comment:
So when we setup the tests, if we want compression type to be 'none' we have to set $rhRun 'compress' to undef, instead of 'none'? But in the MockAllTests we set compress => GZ even though it's really undef via the setup? Anyway, we should discuss so that I can understand better how to use the harness.

Reply:
You can do `undef` which will configure `compress=n` or `none` which will configure `compress-type=none`.  See above for an idea on improving this.

Reply:
Improved as above.